src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -623,6 +624,19 @@ private boolean hasEmptyLineAfter(DetailAST token) {
         return hasEmptyLine(currentEnd + 1, nextBegin - 1);
     }
 
+    /**
+     * Finds first modifier of the token.
+     *
+     * @param token token to check
+     * @return first modifier or {@code null} if there is no modifier
+     */
+    private static DetailAST findFirstModifier(DetailAST token) {
+        return Optional.ofNullable(token)
+            .map(sibling -> sibling.findFirstToken(TokenTypes.MODIFIERS))
+            .map(DetailAST::getFirstChild)
+            .orElse(null);
```

Comment:
does it make sense to avoid null return ?
as we become to use more and more Optional.

Reply:
done